rosdep/base.yaml

```diff
@@ -4857,21 +4855,13 @@ wxwidgets:
   opensuse: [wxGTK-devel]
   rhel: [wxGTK-devel]
   ubuntu:
+    '*': [libwxgtk3.0-dev]
```

Comment:
I just want to note that this is the first time `'*'` is being used. This is going to break users which haven't updated their rosdep version since the new feature has been released.

Reply:
This is actually the second time: https://github.com/ros/rosdistro/pull/17633

If you want we can send an announcement on discourse to let people know that they should upgrade

Reply:
It might be worth a reminder, but an unupgraded rosdep will just fail to resolve the wildcard, entries. rosdep is permissive with extra keys so the old version will just say it can't find a key for the Ubuntu or debian version for that specific key. 0.12 has been out since February so it's been a good amount of time to allow for updating. And the wxwidgets key isn't used in too many places so I think it's a lower priority.